website/types/docusaurus.d.ts

```diff
@@ -0,0 +1,20 @@
+declare module '@docusaurus/*' {
```

Comment:
This file is needed as otherwise I get the following type error in the `Comaprison.tsx` file:
```
Cannot find module '../../../docs/compare-matrix-landing.md' or its corresponding type declarations.

3 import Matrix from '../../../docs/compare-matrix-landing.md';
```